specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2021-01-01/bms.json

```diff
@@ -0,0 +1,10157 @@
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{vaultName}/backupstorageconfig/vaultstorageconfig/prepareDataMove": {
+      "post": {
+        "tags": [
+          "DataMove"
+        ],
+        "description": "Prepares source vault for Data Move operation",
+        "operationId": "BMSPrepareDataMove",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/VaultName"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "description": "Prepare data move request",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/PrepareDataMoveRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "202": {
+            "description": "Accepted"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/NewErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Prepare Data Move": {
+            "$ref": "./examples/BackupDataMove/PrepareDataMove_Post.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{vaultName}/backupstorageconfig/vaultstorageconfig/operationResults/{operationId}": {
+      "get": {
+        "tags": [
+          "BMSPrepareDataMoveOperationResult"
+        ],
+        "description": "Fetches Operation Result for Prepare Data Move",
+        "operationId": "BMSPrepareDataMoveOperationResult_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/VaultName"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "name": "operationId",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/VaultStorageConfigOperationResultResponse"
+            }
+          },
+          "202": {
```

Comment:
If this returns 202 it must be marked as x-ms-long-running-operation (it was previously)

Reply:
This is not meant to be a long-running-operation. Removed the 202 response as well.

Reply:
This is the OperationResults API for the long-running operation above (BMSPrepareDataMove). This returns 202 if the parent BMSPrepareDataMove operation is still in progress, but is not a long running operation in itself.